hack/bare-metal/entry/spire-server-entry-lock-unlock-register.sh

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env bash
+
+#    \\ SPIKE: Secure your secrets with SPIFFE. — https://spike.ist/
+#  \\\\\ Copyright 2024-present SPIKE contributors.
+# \\\\\\\ SPDX-License-Identifier: Apache-2.0
+
+# Set path to the SPIKE binary
+PILOT_PATH="$(pwd)/spike"
+PILOT_SHA=$(sha256sum "$PILOT_PATH" | cut -d' ' -f1)
+
+# Register a new SPIFFE ID for lock/unlock
+spire-server entry create \
+  -spiffeID "spiffe://spike.ist/spike/pilot/role/watchdog" \
+  -parentID "spiffe://spike.ist/spire-agent" \
+  -selector "unix:uid:$(id -u)" \
+  -selector "unix:path:$PILOT_PATH" \
+  -selector "unix:sha256:$PILOT_SHA" \
+  -ttl 3600
```

Comment:
we will check for this SPIFFE ID at the relevant SPIKE Nexus API endpoints; we are not going to check this at SPIKE pilot.